### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by jastinesanchez**
I finished adding the rest of my paragraph the current events section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -34,13 +34,15 @@ <h2>Current events on VR</h2>
         will have the lack of fear of judgement during the session. Without the judgement the patient will be more honest about what they think about
         their body which can help give a more accurate thearpy session.</p>
 
-    <p> (EDIT) Another current event that is happening in Virtual Reality is that there is a new virtual reality software that allows scientist to "walk" inside
+    <p> Another current event that is happening in Virtual Reality is that there is a new virtual reality software that allows scientist to "walk" inside
         cells. With this software this helps the researchs go inside and analyse individual cells, this could be used to understand fundamental problems
         in biology and help develop new treatment for diseases. The software is called vLUME, it was created by scientists at the University of Cambridge
-        and 3D image analysis software company Lume VR Ltd. 
+        and 3D image analysis software company Lume VR Ltd. This allows High Definition microscopy data to be both visualised and analysed in a virtual space.
```
Comment (JeffreyDChau) [inline index.html]: I practiced committing a single.

File: index.html [hunk 2]
```diff
@@ -34,13 +34,15 @@ <h2>Current events on VR</h2>
         will have the lack of fear of judgement during the session. Without the judgement the patient will be more honest about what they think about
         their body which can help give a more accurate thearpy session.</p>
 
-    <p> (EDIT) Another current event that is happening in Virtual Reality is that there is a new virtual reality software that allows scientist to "walk" inside
+    <p> Another current event that is happening in Virtual Reality is that there is a new virtual reality software that allows scientist to "walk" inside
```
Comment (jamesssd) [inline index.html]: Done

File: index.html (hunk 1 above)
Comment (jamesssd) [inline index.html]: Done

File: index.html [hunk 3]
```diff
@@ -34,13 +34,15 @@ <h2>Current events on VR</h2>
...
         cells. With this software this helps the researchs go inside and analyse individual cells, this could be used to understand fundamental problems
         in biology and help develop new treatment for diseases. The software is called vLUME, it was created by scientists at the University of Cambridge
-        and 3D image analysis software company Lume VR Ltd. 
+        and 3D image analysis software company Lume VR Ltd. This allows High Definition microscopy data to be both visualised and analysed in a virtual space.
+        With this technology it can be used to study everything from a single protein to the entire cell as a whole. With super-resolution it makes it possible
+        to obtain images at the nanoscale using tools of physics to get arund the limits imposed by light diffraction. With these tools this has allowed researchers to
+        observe molecular processes as they happen. The only problem is the lack of ways to visualise and analyse it in three dimensions. </p>
```
Comment (jamesssd) [inline index.html]: DOne

File: index.html [hunk 4]
```diff
@@ -34,13 +34,15 @@ <h2>Current events on VR</h2>
...
         
-        https://www.sciencedaily.com/releases/2020/10/201021111605.htm
-        https://www.sciencedaily.com/releases/2020/10/201012120005.htm (edit)
+
```
Comment (jamesssd) [inline index.html]: Done

File: index.html [hunk 5]
```diff
@@ -59,55 +61,57 @@ <h3> Sources </h3>
...
+
+    
+
+  <h2>Current events on VR</h2>
+    /*
+    Any new tech thats been released recently and how VR is now.
+    */
+    
+<h2>Cost of a VR headset right now</h2>
```
Comment (jamesssd) [inline index.html]: Done

File: index.html [hunk 6]
```diff
@@ -59,55 +61,57 @@ <h3> Sources </h3>
...
+        <li>Samsung Gear VR: $129</li>
+    </ol>    
+/*
+    How much stipend it will cost for us to get a VR set now
+    */
+    
+<h2>Potential future VR technology</h2>
+    /*
+    Any ideas for new technology
+    */
```
Comment (jamesssd) [inline index.html]: Done

**Review by jamesssd — APPROVED** [review]

**Pull request merged**